spec/models/user_spec.rb

```diff
@@ -7,7 +7,7 @@
   end
 
   describe "associations" do
-    it { should have_and_belong_to_many(:organisations) }
+    it { should belong_to(:organisation).optional(true) }
```

Comment:
We might want to change these to:

`it { is_expected.to belong_to ... }` (this works because the old "should" syntax is still enabled in the configuration)

Reply:
Oh, I thought it was syntax from the `shoulda-matchers` gem and didn't think much more about it. I'll give it a try.

Reply:
Yeah this worked!